ddtrace/tracer/rules_sampler.go

```diff
@@ -521,10 +553,13 @@ func unmarshalSamplingRules(b []byte, spanType SamplingRuleType) ([]SamplingRule
 		return nil, nil
 	}
 	var jsonRules []struct {
-		Service      string      `json:"service"`
-		Name         string      `json:"name"`
-		Rate         json.Number `json:"sample_rate"`
-		MaxPerSecond float64     `json:"max_per_second"`
+		Service      string            `json:"service"`
+		Name         string            `json:"name"`
+		Rate         json.Number       `json:"sample_rate"`
+		MaxPerSecond float64           `json:"max_per_second"`
+		Resource     string            `json:"resource"`
+		TargetRoot   string            `json:"target_span"`
```

Comment:
```suggestion
		TargetSpan   string            `json:"target_span"`
```
I believe `TargetSpan` would be more accurate here since the field is a string, not a boolean.